<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_queue</class>
 <widget class="QDialog" name="dlg_queue">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Common background widget style */
QWidget {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(240, 245, 235, 1), stop:0.5 rgba(255, 255, 255, 1), stop:1 rgba(240, 245, 235, 1)); /* Gradient from light green to white to light green */
    color: #333333; /* Dark Grey */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

/* Buttons common style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #E0E0E0, stop:1 #FFFFFF); /* Light Grey to White Gradient */
    color: #333333; /* Dark Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
    border-radius: 4px;
    padding: 5px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

QPushButton:hover {
    background-color: #D6D6D6; /* Slightly Darker Grey */
}

QPushButton:pressed {
    background-color: #BFBFBF; /* Darker Grey */
}

</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>311</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #FFFFFF, stop:1 #F0F0F0); /* White to Very Light Grey Gradient */
    color: #333333; /* Dark Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>120</y>
     <width>321</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #FFFFFF, stop:1 #F0F0F0); /* White to Very Light Grey Gradient */
    color: #333333; /* Dark Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>321</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #FFFFFF, stop:1 #F0F0F0); /* White to Very Light Grey Gradient */
    color: #333333; /* Dark Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>570</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
}
</string>
   </property>
   <property name="text">
    <string>4人桌</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>570</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
}
</string>
   </property>
   <property name="text">
    <string>8人圆桌</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Common background widget style */
QWidget {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(240, 245, 235, 1), stop:0.5 rgba(255, 255, 255, 1), stop:1 rgba(240, 245, 235, 1)); /* Gradient from light green to white to light green */
    color: #333333; /* Dark Grey */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}
</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Common button style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 170, 127, 1), stop:1 rgba(100, 190, 145, 1)); /* Gradient from primary to slightly lighter green */
    color: #FFFFFF; /* White text for contrast */
    border: 1px solid #5DAE89; /* Darker green border */
    border-radius: 8px; /* Slightly more rounded corners */
    padding: 6px 12px; /* Increased padding for a more comfortable touch target */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
    font-weight: bold; /* Bold text for emphasis */
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 190, 145, 1), stop:1 rgba(85, 170, 127, 1)); /* Reverse gradient for hover effect */
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(70, 150, 110, 1), stop:1 rgba(85, 170, 127, 1)); /* Darker gradient for pressed effect */
}
</string>
    </property>
    <property name="text">
     <string>排队</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_seat">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Common button style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 170, 127, 1), stop:1 rgba(100, 190, 145, 1)); /* Gradient from primary to slightly lighter green */
    color: #FFFFFF; /* White text for contrast */
    border: 1px solid #5DAE89; /* Darker green border */
    border-radius: 8px; /* Slightly more rounded corners */
    padding: 6px 12px; /* Increased padding for a more comfortable touch target */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
    font-weight: bold; /* Bold text for emphasis */
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 190, 145, 1), stop:1 rgba(85, 170, 127, 1)); /* Reverse gradient for hover effect */
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(70, 150, 110, 1), stop:1 rgba(85, 170, 127, 1)); /* Darker gradient for pressed effect */
}
</string>
    </property>
    <property name="text">
     <string>快速入座</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>599</y>
     <width>1131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(35, 70, 104);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>570</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: transparent;
}
</string>
   </property>
   <property name="text">
    <string>2人桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>88</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_q1seat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>入座</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_q2seat">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>入座</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_q3seat">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>入座</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
